### ISSUE TITLE
Mouse not rendering over adobe photoshop / exel

### ISSUE THREAD

**Issue by ghost** [cedrozor/myrtille#325]
Hi, i tried to work in photoshop truth myrtille, but mouse cursor not rendering properly.
Photoshop:
![image](https://user-images.githubusercontent.com/20506433/144744633-a0a6beec-7066-4277-b55d-d71bc0df5bdf.png)

Also in Excel same problem:

![9rnxEP9pll](https://user-images.githubusercontent.com/20506433/144745062-00d724c3-b130-4886-8cb2-91c2e2ed29a3.gif)

Over classical RDP all working fine

**Comment by cedrozor** [comment]
Yes, there are mouse cursor issues on some applications (see https://github.com/cedrozor/myrtille/issues/161). Unfortunately, I never found what is the problem exactly :/ you can try to disable the mouse cursor shadow server side (https://www.thewindowsclub.com/enable-disable-mouse-pointer-shadow-windows), it proved to work sometimes

**Comment by thoughtspacewebsites** [comment]
I'm experiencing this same issue in Ubuntu clients as well.It's pretty bad within Affinity Designer and makes it pretty hard to use. Of interesting note, however, is that I also have Fusion 360 up and running, and it seems to work just fine with cursors. Almost seems like it's only certain cursors that do this. I saw you mentioned this seems to be a Linux specific problem, could it have to do with certain themes missing certain cursors? Not sure how this works, just trying to think of a possible solution as I'd really like to keep using Myrtille but this is definitely a big deal and makes apps pretty hard to use.

**Comment by thoughtspacewebsites** [comment]
Also, mouse cursor shadow did not seem to help in my instance :/

**Comment by cedrozor** [comment]
It may indeed be due to some cursors unavailable on the RDP client side (the Myrtille machine). What you can do is check for `.cur` files into your application(s) path(s) and copy them into `C:\Windows\Cursors` (assuming C: is the system drive) on the Myrtille machine.

See https://www.howtogeek.com/664962/how-to-change-your-mouse-cursor-theme-on-windows-10/ for more details with custom cursors and cursors themes.

**Comment by thoughtspacewebsites** [comment]
Trying to look into this more, and I have some confusion. Where would I go about locating .cur files for my application? Is there a normal place they're stored in? Also, I just have to make a copy of them in Windows\Cursors, and myrtille will pass them client side? My client is Ubuntu, and at first I thought you were saying I would have to download the cursors to my Ubuntu machine, but it sounds like I just need to copy them to the correct directory on the server in this case. Can you please give me more insight?

**Comment by cedrozor** [comment]
Indeed, by client I meant the machine running Myrtille, not the end user machine (running the browser). The mouse cursor `.cur` data is streamed into the browser, then updated there. Your application either use custom cursors, only present into the application path, or into `C:\Windows\Cursors`.

First search for `.cur` files into your application path on the remote server. If there are none, copy the whole `C:\Windows\Cursors` folder from the remote server to the Myrtille machine, same location (make a backup first).

**Comment by dbychkov** [comment]
@cedrozor In my case I have Myrtille service and Excel both installed on the same machine/server. Could you please advise what needs to copied where?

**Comment by AffinityUser** [comment]
@cedrozor - Hello. We too are experiencing this issue of a blurry cursor when we run Myrtille on Windows Server 2022. The issue does not occur on when we run Myrtille on Windows Server 2012 or Windows Server 2019 servers -  it only happens on our Windows Server 2022 boxes. Any suggestions will be greatly appreciated. 

https://github.com/cedrozor/myrtille/assets/170148408/7e1351f7-90d6-4f19-988a-7eead0bbf0bc
